internal/pkg/bootstrap/logging/factory.go

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package logging
+
+import (
+	"github.com/edgexfoundry/edgex-go/internal/pkg/bootstrap/interfaces"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/models"
+)
+
+// FactoryToStdout returns a logger.LoggingClient that outputs to stdout.
+func FactoryToStdout() logger.LoggingClient {
+	return logger.NewClient(clients.CoreDataServiceKey, false, "", models.DebugLog)
```

Comment:
I think the service key needs to be passed in as a parameter here rather than hard-coded. This function is called from the generic bootstrap/Run() function and will thus be utilized by any service.

Reply:
You're right.  I missed converting this reference.